package.json

```diff
@@ -77,6 +77,7 @@
         "typescript": "^4.6.3"
     },
     "dependencies": {
+        "node-domexception": "^1.0.0",
```

Comment:
I don't prefer any other dependency actually. But if we have to then I am ok with that.

Reply:
this simply forces exposing of node-domexception, so if we want to support node version 14-17  [domexception was added in node 17]

Reply:
What is the situation for v12? We need to support >=v12

Reply:
eventtarget doesnt exist in node <14

Reply:
Node 12 went end-of-life more than one year ago, support could be removed (even node 14 is not maintained anymore), actually removing node 12 would make it possible to clean up the legacy NAPI thread-safe callback mess.